_posts/en/2021-08-20-micro-acquisition-pilot-update2.md

```diff
@@ -0,0 +1,50 @@
+ref: micro-acquisition-update2
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team and Micro-Acquisition Pilot team members"
+date: "2021-08-20"
+excerpt: "The Micro-Acquisition pilot is live! In this blog post we share what we have learned so far and how we are adjusting and improving."
+---
+<!--markdownlint-disable MD033-->
+Since our last status update, the [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html) went live and the [first opportunity](https://micro-acquisition.alpha.canada.ca/2021/06/23/building-custom-Azure-marketplace-policies.html) was posted.
+
+As part of the Micro-Acquisition pilot we are trying things, learning and adjusting as we go.
+In preparation for the next opportunity we have made a few changes based on what we learned from the first opportunity.
+
+## Added a mailing list
+
+When we didn't get a lot of followers on our [Twitter account](https://twitter.com/microbuysgc), we did some testing and realized 2 things:
+
+1. It wasn't clear what information would be found by following the Twitter account and
+2. A lot of our target audience is not on Twitter
+
+We now have a mailing list that anyone can subscribe to.
+Through the [mailing list](https://forms-formulaires.alpha.canada.ca/id/36) (and Twitter) we will be sharing when new opportunities are posted and/or updated. We also updated the text on the opportunities page to be more clear what information is available via the Twitter account and mailing list.
+
+## Updated evaluation question
+
+Our first opportunity was for a less well known language (Powershell).
+We realized from the applications we received that there were people who likely could do the work but the phrasing of our evaluation criteria prevented us from giving them a pass because they didn't have experience specifically in Powershell.
+Where possible, future opportunities will see the evaluation criteria change from:
+> Please provide a short written statement demonstrating how you have the required skills in x
+
+to
+
+> Please provide a short written statement demonstrating how you have the required skills in x or a similar language.
+
+We've also added [example answers which would get a passing mark](https://micro-acquisition.alpha.canada.ca/en/user-guide.html#example-answers-that-would-get-a-passing-mark) to the User Guide.
+
+## Using GC Forms for the application form
+
+We are replacing our pdf application form with a form on [GC Forms](https://forms-formulaires.alpha.canada.ca/en/welcome-bienvenue).
+GC Forms is a CDS product and it is accessible, secure and has full authority to operate.
+GC Forms is a service - you tell CDS your need, they build your form, test it and then send you the link.
+
+IT IS GREAT.
+
+We are excited because not only will we no longer have to maintain a pdf form, but user experience for suppliers will be improved as the form will tell them if they have missed filling something in.
```

Comment:
I think mentioning the Accessibility, Privacy and Security could good  as well :)

Reply:
I think I've got this in the reworked text but let me know if you think I could/should add more. :)

Reply:
This version has been sent to translation. Will be back on Sept. 13.